packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss

```diff
@@ -12,9 +12,12 @@ $clr-combobox-trigger-left-padding: $clr_baselineRem_0_25 !default;
 $clr-combobox-trigger-right-padding: $clr_baselineRem_1 !default;
 
 $clr-combobox-font-size: $clr-p2-font-size !default;
+$clr-combobox-border-color: hsl(198, 10%, 90%) !default;
+$clr-combobox-border-radius: $clr_baselineRem_3px !default;
 
-$clr-combobox-input-background-color: hsl(201, 10%, 96%) !default;
-$clr-combobox-pill-background-color: hsl(201, 80%, 95%) !default;
-$clr-combobox-pill-border-color: hsl(201, 55%, 81%) !default;
+$clr-combobox-input-background-color: hsl(198, 12%, 97%) !default;
+$clr-combobox-pill-background-color: hsl(198, 0%, 100%) !default;
+$clr-combobox-pill-border-color: hsl(198, 10%, 90%) !default;
```

Comment:
Why is design giving us so many off-palette colors?

Reply:
After pairing with design on the problem, we changed all these to Clarity palette colors ($clr-color-..............).
Colors in the dark theme were updated to match those from the Accordion component, which features similar levels of shading as the Combobox. We don't have pre-defined dark theme palette, but this should be addressed separately from this issue.
Caret color was updated to text color, for better color contrast.